README.md

```diff
@@ -1,158 +1,16 @@
-# chart-verifier
+# `chart-verifier`: Rules based tool to certify Helm Charts
```

Comment:
Why back ticks? On line 5 you use: **chart-verifier** IMHO that makes sense. If you based it on odo - odo is a command and they want to use the command as the name of the tool. In this case let us distinguish between the name of the tool and the command, using Bold and backticks/monospace respectively.